app/src/renderer/components/common/NiModalWelcome.vue

```diff
@@ -0,0 +1,158 @@
+<template lang="pug">
+.ni-modal-welcome(v-if="active")
+
+  .ni-mw(v-if="status == 'welcome'"): .ni-mw-container
+    .ni-mw-header
+      .ni-mw-title Welcome to Cosmos!
+    .ni-mw-main
+      list-item(@click.native="setStatus('new-wallet')" title="Create new wallet") 
+      list-item(@click.native="setStatus('existing-wallet')" title="Sign in with existing wallet")
```

Comment:
is there a way we can differentiate between "sign in with existing wallet" and "sign in with hardware wallet"? in both cases, you are signing in with an existing wallet. one is explaining the status of the account while the other is describing the medium.

perhaps "sign in with mnemonic" or "sign in with password" and "sign in with hardware wallet" because both of these describe the medium.

Reply:
i'm also not sold on "create new wallet" yet. the wallet is a feature of the cosmos-ui. creating a new "wallet" provides access to the UI — not just the wallet.

technically, when you click "create new wallet" are we generating a new public key for them? this would be a lot like an account. 

once they create a new one — or if they have an existing one — they can log out and then create another. wouldn't this address #23 ?